php/class-wp-cli.php

```diff
@@ -25,7 +25,7 @@ public function create_guest_authors( $args, $assoc_args ) {
 			);
 		$this->args = wp_parse_args( $assoc_args, $defaults );
 
-		$users = get_users();
+		$users = get_users( array( 'role__in' => array( 'administrator', 'editor', 'author' ) ) );
```

Comment:
I think we may want to allow filtering of this in some way. Some sites may have other roles they want to be allowed to be added as coauthors. Since it's a CLI command we may want to allow an optional argument to overwrite the default list of roles :)

Also, I think the `contributor` role should be included by default, as it has the `edit_posts` cap and thus permission to be added as coauthor :)

Reply:
Works for me. After lunch I'll put together some thoughts on a --roles flag :)